templates/conf-mesos-slave.j2

```diff
@@ -5,6 +5,7 @@ export MESOS_CONTAINERIZERS="{{ mesos_containerizers }}"
 export MESOS_EXECUTOR_REGISTRATION_TIMEOUT="{{ mesos_executor_timeout }}"
 export MESOS_PORT="{{ mesos_slave_port }}"
 export MESOS_WORK_DIR="{{ mesos_work_dir }}"
+export MESOS_ATTRIBUTES="node_type:{{ mesos_node_type }}"
```

Comment:
Setting attributes is currently done using:
```
mesos_slave_additional_configs:
    - name: ATTRIBUTES
      value: '"rack:external"'
```

Does this change break this original behavior?